src/auro-toast.js

```diff
@@ -80,16 +90,24 @@ export class AuroToast extends LitElement {
 
   /**
    * @private
-   * @param {string} svgContent - The imported svg icon.
+   * @param {string} category - The category of the icon.
+   * @param {string} name - The name of the icon.
    * @returns {string} - The html template for the icon.
    */
-  generateIconHtml(svgContent) {
-    const dom = new DOMParser().parseFromString(svgContent, 'text/html'),
-      svg = dom.body.firstChild;
-
+  generateIconHtml(category, name) {
     return this.noIcon
       ? html``
-      : html`<div class="icon">${svg}</div>`;
+      : html`
+        <${this.iconTag} 
+          ?onDark="${!this.variant}" 
+          ?error="${this.getAttribute('variant') === 'error'}" 
+          ?success="${this.getAttribute('variant') === 'success'}" 
+          class="icon" 
+          customSize 
+          category="${category}" 
+          name="${name}">
+        </${this.iconTag}>
+      `;
```

Comment:
We should not be using CDN resources inside another element. Please update to support direct dependencies of these icons.